util/src/test/java/io/grpc/util/ForwardingLoadBalancerTest.java

```diff
@@ -45,6 +45,8 @@ public void allMethodsForwarded() throws Exception {
         mockDelegate,
         new TestBalancer(),
         Arrays.asList(
-            LoadBalancer.class.getMethod("acceptResolvedAddresses", ResolvedAddresses.class)));
+            LoadBalancer.class.getMethod("acceptResolvedAddresses", ResolvedAddresses.class),
```

Comment:
acceptResolvedAddresses should be removed already with the current code. After you change handleResolverAddresses to forward, it should be removed as well.